Tests/PlotTests/HTMLTests.swift

```diff
@@ -602,6 +602,13 @@ final class HTMLTests: XCTestCase {
         assertEqualHTMLContent(html, "<body><noscript>NoScript</noscript></body>")
     }
 
+    func testDivWithTitleAttribute() {
+        let html = HTML(.body(.div("Element"), .title("Title")))
```

Comment:
The expected output of this expression is one `div` and one `title` element as siblings. I'm guessing that you'd like to do this instead:
```swift
HTML(.body(.div("Element", .title("Title"))))
```
Since attributes are applied to the element that they appear within.

There is already an API for the `title` attribute, but it's currently constrained to `HTML.AbbreviationContext` (see [this file](https://github.com/JohnSundell/Plot/blob/master/Sources/Plot/API/HTMLAttributes.swift#L480)), so perhaps a solution would be to change that constraint to `HTML.BodyContext` instead? If we want the `title` attribute to be applicable to all `<body>` elements, that is.

Reply:
> I'm guessing that you'd like to do this instead:

😳 You're absolutely right... That's the problem with committing code in a test that doesn't compile... It's really hard to know whether that code is wrong because it's not written yet, or because it's just wrong 😂